packages/teleterm/src/ui/services/quickInput/quickPickers.ts

```diff
@@ -101,6 +104,10 @@ export class QuickLoginPicker extends ClusterPicker {
     return items;
   }
 
+  getAutocompleteResult() {
+    return null;
+  }
+
```

Comment:
I left some of the old pickers in the code, they are not doing any harm ATM and we might just remove them in the future.